python/clidef.py

```diff
@@ -71,6 +71,10 @@ class Prefix6Handler(PrefixBase):
     argtype = 'const struct prefix_ipv6 *'
     decl = Template('struct prefix_ipv6 $varname = { };')
     code = Template('_fail = !str2prefix_ipv6(argv[_i]->arg, &$varname);')
+class PrefixEthHandler(PrefixBase):
```

Comment:
if you want `<A.B.C.D/M|M:A:C/M>` to merge into `struct prefix`, you also need to touch this above:
```
    def combine(self, other):
        if type(self) == type(other):
            return other
        if type(other) in [Prefix4Handler, Prefix6Handler, PrefixGenHandler]:    # <----- here
            return PrefixGenHandler(None)
```